wishlist/urls.py

```diff
@@ -3,7 +3,13 @@
 
 urlpatterns = [
     path('wishlist/', views.wishlist, name='wishlist'),
-    path('wishlist/d/<int:book_id>', views.deleteItem, name='deleteItem2'),
+    path('wishlist/d/<int:book_id>', views.deleteItem, name='deleteItem'),
```

Comment:
I believe this is the line that was conflicting with the cart page due to them both delete buttons having the same name. Please update the name.

Reply:
changed it